sparkql/fields/base.py

```diff
@@ -206,6 +213,8 @@ class AtomicField(BaseField):
     ```
     """
 
+    __hash__ = BaseField.__hash__
```

Comment:
If a class overrides `__eq__`, it disables `__hash__`.

Reply:
I discovered it because of unit tests failing. I didn't know myself either. I missed those two. Sorry about that. Fixed now.